build/common.jl

```diff
@@ -64,7 +64,7 @@ end
 
 const REPO_PATH = abspath(joinpath(@__DIR__, ".."))
 const REPO_HTTPS_URL = convert_to_https_url(remote_url(REPO_PATH))
-const COMPILED_DIR = readlink(joinpath(REPO_PATH, "build", "bazel-bin"))
+const COMPILED_DIR = joinpath(REPO_PATH, "build", "bazel-bin")
```

Comment:
I'll just call out that the reason we use `readlink` in the first place is because `Tar.create` seemingly can't build against symbolic links, so you need pass it the hardpath.

Reply:
Thanks. I'll be sure to do additional testing. It is working as is on Linux CI